etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/ConvertToOmlOrder.java

```diff
@@ -9,8 +9,8 @@
 import java.util.Map;
 import org.hl7.fhir.r4.model.Bundle;
 
-/** Custom transformation to convert an ORM order to OML. */
-public class convertToOmlOrder implements CustomFhirTransformation {
+/** Converts an ORM order to OML. */
+public class ConvertToOmlOrder implements CustomFhirTransformation {
```

Comment:
Should the `transform` method have a `try`/`catch` and thrown a `RuleExecutionException` like the newer transformations?  And then unit tests for this too.

Reply:
Yes, I plan to add them in another PR